app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1317,23 +1315,28 @@ class PatientRepositoryAndroidTest {
         creationFacilityUuid = facilityUuid,
         status = Scheduled
     )
-    val recentPatient2 = savePatientWithAppointment(
-        facilityUuid = facilityUuid,
-        creationFacilityUuid = facilityUuid,
-        status = Cancelled
-    )
-    val recentPatient3 = savePatientWithAppointment(
-        facilityUuid = facilityUuid,
-        creationFacilityUuid = facilityUuid,
-        status = Visited
-    )
 
     val recentPatients = patientRepository
         .recentPatients(facilityUuid)
         .blockingFirst()
     assertThat(recentPatients).isEqualTo(listOf(recentPatient1))
   }
 
+  @Test
+  fun verify_only_list_of_colonies_or_villages_are_received_from_patient_address() {
+    val address1 = testData.patientAddress(colonyOrVillage = "Sharma Courts")
+    val address2 = testData.patientAddress(colonyOrVillage = "7111 Jain Mountains")
+    val address3 = testData.patientAddress(colonyOrVillage = "138, residency road")
```

Comment:
We should ideally also create duplicate addresses and ensure that final set of results are distinct and have no repeating elements.